setup.py

```diff
@@ -1,39 +1,67 @@
-from setuptools import setup, find_packages
+"""setuptools config for nowplaypadgen"""
+
 import os
 import re
+import sys
+
+import nowplaypadgen
+
+from setuptools import setup, find_packages
+
 
 def readme():
-    with open('README.rst') as f:
-        return f.read()
+    """Load README.rst file for description"""
+    with open('README.rst') as readme_file:
+        return readme_file.read()
+
 
 def get_version():
-    VERSIONFILE = os.path.join('nowplaypadgen', '__init__.py')
-    initfile_lines = open(VERSIONFILE, 'rt').readlines()
-    VSRE = r"^__version__ = ['\"]([^'\"]*)['\"]"
+    """Get version from main module"""
+    version_file = os.path.join('nowplaypadgen', '__init__.py')
+    initfile_lines = open(version_file, 'rt').readlines()
     for line in initfile_lines:
-        mo = re.search(VSRE, line, re.M)
-        if mo:
-            return mo.group(1)
-    raise RuntimeError('Unable to find version string in %s.' % VERSIONFILE)
+        matches = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", line, re.M)
+        if matches:
+            return matches.group(1)
+    raise RuntimeError('Unable to find version string in %s.' % version_file)
+
+
+def pytest_runner():
+    """Require pytest-runner for invocations of setup.py that will invoke it."""
+    needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
+    return ['pytest-runner~=2.11.1'] if needs_pytest else []
+
+INSTALL_REQUIRES = [
+    'pytz>=2017.02',
+    'setuptools~=28.8.0',
+    'markdown',
+] + pytest_runner()
+
+TESTS_REQUIRE = [
+    'pytest~=3.2.1',
+    'pytest-pylint~=0.7.1'
+]
+
+ENTRY_POINTS = {
+    'console_scripts': [
+        'nowplay-padgen = nowplaypadgen.__main__:main'
+    ]
+}
 
 setup(
     name='nowplaypadgen',
     version=get_version(),
-    description='DAB+ now playing PAD (DLS+ and MOT SLS) generator',
+    description=nowplaypadgen.__doc__,
```

Comment:
Is this safe to do? Don't we have a potential race import condition here, similar as what `get_version()` tries to avoid? This is also related to the comment above regarding `import nowplaypadgen`.

Reply:
IMHO my change here is hard-core pythonic.

Reply:
This is prone to the same import problem as [above](https://github.com/radiorabe/nowplaypadgen/pull/1#discussion_r136910362), so I doubt that this is the pythonic way to do it, or do you have a reference? To avoid doc redundancy within `long_description ` we could use the suggestions from the [Python Packaging User Guide's](https://packaging.python.org/) [sampleproject - `setup.py`](https://github.com/pypa/sampleproject/blob/02130aeda025ca86975258f953b5d2531d74e94c/setup.py#L16-L18)

Reply:
The `long_description` is already being set up as per the linked article. Se my comment on the import for why I believe tis to be sane.

Reply:
See my [comment above](https://github.com/radiorabe/nowplaypadgen/pull/1#discussion_r137509440) why I think this is an actual issue.